llsd/base.py

```diff
@@ -366,13 +353,22 @@ def __init__(self):
         }
 
 
+_X_ORD = ord(b'x')
+_BACKSLASH_ORD = ord(b'\\')
```

Comment:
`b'x'[0]` is normally the construct I use in Python 3 because a compiler pass can optimize it away:

```
$ python3 -c 'import dis; foo = lambda: ord(b"x"); dis.dis(foo)'
  1           0 LOAD_GLOBAL              0 (ord)
              2 LOAD_CONST               1 (b'x')
              4 CALL_FUNCTION            1
              6 RETURN_VALUE
$ python3 -c 'import dis; foo = lambda: b"x"[0]; dis.dis(foo)'
  1           0 LOAD_CONST               1 (120)
              2 RETURN_VALUE
```

however it doesn't work in Python2 due to the `str`-ness of the `bytes` object:

```python
>>> bytes(b"x")[0]
'x'
>>> from llsd.base import B
>>> B(b"x")[0]
'x'
```

and you must use `ord()`